pom.xml

```diff
@@ -4,7 +4,7 @@
   <groupId>com.google.cloud</groupId>
   <artifactId>google-cloud-spanner-parent</artifactId>
   <packaging>pom</packaging>
-  <version>6.23.2-SNAPSHOT</version><!-- {x-version-update:google-cloud-spanner:current} -->
+  <version>6.23.2-SNAPSHOT</version>
```

Comment:
Why did we remove the metadata here?

Reply:
We need not remove this. Re-added the same.